src/unit_tests/CMakeLists.txt

```diff
@@ -107,7 +107,7 @@ add_custom_target(coverage
 
     # Add baseline counters
     COMMAND ${LCOV_PATH} -q --gcov-tool ${GCOV_PATH} -a coverage.base -a coverage.info --rc lcov_branch_coverage=1 --output-file coverage.total
-    COMMAND ${LCOV_PATH} --gcov-tool ${GCOV_PATH} --remove coverage.total "*external/*" --remove coverage.total "*unit_tests/*" --rc lcov_branch_coverage=1 --output-file coverage.info.cleaned
+    COMMAND ${LCOV_PATH} --gcov-tool ${GCOV_PATH} --remove coverage.total "*external/*" --remove coverage.total "*unit_tests/*" --remove coverage.total "*data_provider/*" --remove coverage.total "*shared_modules/*" --remove coverage.total "*syscollector*" --rc lcov_branch_coverage=1 --output-file coverage.info.cleaned
```

Comment:
should this be "*wazuh_modules/syscollector/*" ? what about the analysisd/decoders/syscollector.c file?

Reply:
This excludes folders instead of files. So, syscollector works fine.

Reply:
I agree, we should only exclude syscollector module and not the decoder.